odoo/models.py

```diff
@@ -1895,7 +1895,7 @@ def _read_group_fill_temporal(self, data, groupby, aggregated_fields, annotated_
     def _read_group_prepare(self, orderby, aggregated_fields, annotated_groupbys, query):
         """
         Prepares the GROUP BY and ORDER BY terms for the read_group method. Adds the missing JOIN clause
-        to the query if order should be computed against m2o field. 
+        to the query if order should be computed against m2o field.
```

Comment:
Useless diff, could you remove it, to keep a clean history on models.py ? 

It's important to avoid noise on the basic files.